src/containerapp/azext_containerapp/commands.py

```diff
@@ -32,6 +32,7 @@ def load_command_table(self, _):
     with self.command_group('containerapp replica') as g:
         g.custom_show_command('show', 'get_replica')  # TODO implement the table transformer
         g.custom_command('list', 'list_replicas')
+        g.custom_command('count', 'count_replicas', is_preview=True)
```

Comment:
>is_preview=True [](http://example.com/codeflow?start=52&length=15)

why is this preview ?

Reply:
Hi @pagariyaalok 

> why is this preview ?

We have containerapp in core https://github.com/Azure/azure-cli/tree/dev/src/azure-cli/azure/cli/command_modules/containerapp, we only put GA commands in core without `is_preview=True`.
About the iteration process for new commands:
Firstly, add new commands or new features in containerapp extension with tag `is_preview=True`.
After released in extension and being mature enough, you can move it to the core, at that time you can remove the `is_preview=True` in extension.

Reply:
Thank you, Xinyu!